tests/database/query/test-post-query-builder.php

```diff
@@ -240,32 +240,32 @@ public function test_query_by_author_name() {
 		$this->assertEquals( $post_id, $first->id() );
 	}
 
-	public function test_query_with_multiple() {
-		$post_a = $this->get_random_post_id();
-		$post_b = $this->get_random_post_id( [ 'post_type' => Another_Testable_Post::get_object_name() ] );
+	// public function test_query_with_multiple() {
```

Comment:
Could you just remove this function instead of commenting it out?

Reply:
I actually plan on fixing this test in #132!